src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,62 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
+    }
+
+    public Manufacturer(Long id, String name, String country) {
+        this.id = id;
+        this.name = name;
+        this.country = country;
```

Comment:
```suggestion
        this(name,country);
```

it is better to use one constructor into another

Reply:
here